Language/Reference/User-Interface-Help/references-dialog-box.md

```diff
@@ -23,7 +23,7 @@ The following table describes the dialog box options.
 |**Available References**|Lists the references available to your project.<br/><br/>After you set a reference to an object library by selecting the check box next to its name, you can find a specific object and its methods and properties in the [Object Browser](../../Glossary/vbe-glossary.md#object-browser).<br/><br/>If you are not using any objects in a referenced library, you should clear the check box for that reference to minimize the number of object references Visual Basic must resolve, thus reducing the time it takes your project to compile. You can't remove a reference for an item that is used in your project.<br/><br/>If you remove a reference to an object that you are currently using in your project, you'll receive an error the next time you refer to that object.<br/><br/>References not in use are listed alphabetically.<br/><br/>**NOTE**: You can't remove the "Visual Basic For Applications" and "Visual Basic objects and procedures" references because they are necessary for running Visual Basic.|
 |**Priority** buttons |Move references up ![Move up](../../../images/tbr_pri1_ZA01201723.gif) and down ![Move down](../../../images/tbr_pri2_ZA01201724.gif) on the list.<br/><br/>When you refer to an object in code, Visual Basic searches each referenced library selected in the **References** dialog box in the order that the libraries are displayed. If two referenced libraries contain objects with the same name, Visual Basic uses the definition provided by the library listed higher in the **Available References** box.|
 |**Result**|Displays the name and path of the reference selected in the **Available References** box, as well as the language version.|
-|**Browse**|Displays the **Add References** dialog box so that you can search other directories for and add references to the **Available References** box for the following types:<br/><br/>- [Type Libraries](../../Glossary/vbe-glossary.md#type-library) (\*.olb, \*.tlb, \*.dll)<br/>- [Executable Files](../../Glossary/vbe-glossary.md#executable-file) (\*.exe, \*.dll)<br/>- ActiveX Controls (\*.ocx)<br/>- All Files (\*.\*)<br/><br/>The **Add References** dialog box is the **Open** common dialog box.|
+|**Browse**|Displays the **Add References** dialog box so that you can search other directories for and add references to the **Available References** box for the following types:<br/><br/>- Application-specific files (Excel: xlsm, xlam. PowerPoint: ppam. Word: docm, dotm.)<br/>- [Type Libraries](../../Glossary/vbe-glossary.md#type-library) (\*.olb, \*.tlb, \*.dll)<br/>- [Executable Files](../../Glossary/vbe-glossary.md#executable-file) (\*.exe, \*.dll)<br/>- ActiveX Controls (\*.ocx)<br/>- All Files (\*.\*)<br/><br/>The **Add References** dialog box is the **Open** common dialog box.|
```

Comment:
```suggestion
|**Browse**|Displays the **Add References** dialog box so that you can search other directories for and add references to the **Available References** box for the following file types:<br/><br/>- Application-specific files for Excel (.xlsm, .xlam), PowerPoint ( .ppam), and Word (.docm, .dotm)<br/>- [Type Libraries](../../Glossary/vbe-glossary.md#type-library) (\*.olb, \*.tlb, \*.dll)<br/>- [Executable Files](../../Glossary/vbe-glossary.md#executable-file) (\*.exe, \*.dll)<br/>- ActiveX Controls (\*.ocx)<br/>- All Files (\*.\*)<br/><br/>The **Add References** dialog box is the **Open** common dialog box.|
```